metrics/registry.lua

```diff
@@ -70,6 +70,14 @@ function Registry:register_callback(callback)
     end
 end
 
+function Registry:unregister_callback(callback)
+    for i, registered_callback in ipairs(self.callbacks) do
+        if registered_callback == callback then
+            table.remove(self.callbacks, i)
+        end
+    end
+end
+
 function Registry:set_labels(label_pairs)
     self.label_pairs = table.copy(label_pairs)
```

Comment:
may be label_pairs or {} since this method may be called straight from user code?

Reply:
That method not in metrics API and used only in `set_global_labels`.
https://github.com/tarantool/metrics/blob/9c41c9fcab21e048bf96da91d11e26ed183ee7e3/metrics/init.lua#L86